### DISCUSSION TITLE
Finding mitool commands

### DISCUSSION THREAD

**Discussion by racng** [milaboratory/mixcr#1814] [Q&A] [answered]
How can I run mitool commands (mitool parse, mitool filter, etc.)? They look incredibly useful!
I tried installing mitools from your github but it looks like a completely different package.
Thanks in advance!

**Comment by mizraelson** [comment, marked as answer]
Hi, could you please email me on support@milaboratories.com? The latest version of MiTool is not publicly available anymore.
